### PR TITLE
Add method to do post requests in browsershot

### PR DESCRIPTION
**Pull request by tomhatzer**
First version of a solution for issue https://github.com/spatie/browsershot/issues/214

### CODE REVIEW

File: tests/BrowsershotTest.php [hunk 1]
```diff
@@ -910,4 +910,24 @@ public function it_can_add_a_wait_for_function_to_puppeteer()
...
     }
+
+    /** @test **/
+    public function it_can_send_post_parameters_and_get_the_body_html()
+    {
+        $html = Browsershot::url('http://example.com')
+            ->usePost(['hello' => 'hi there!', 'is it me' => 'you are looking for?'])
+            ->bodyHtml();
+
+        $this->assertContains('<h1>Example Domain</h1>', $html);
```
Comment (freekmurze) [inline tests/BrowsershotTest.php]: How does this output prove that `usePost` is actually working?

**Comment by freekmurze** [comment]
Thank you for this, I'll do a thorough review soon.

File: tests/BrowsershotTest.php (hunk 1 above)
Comment (tomhatzer) [reply tests/BrowsershotTest.php]: There's currently no way of proving this using example.com as domain. We'd have to use something else like httpbin.org or another http post service that returns the post values. If that's possible I'll customize the 2 tests.

**Comment by freekmurze** [comment]
@delia-curiel reported in #214 that this code is not working. 

@tomhatzer could you take a look at this?

**Comment by tomhatzer** [comment]
I'll check this again. Posting data works, but that website has some header checks or something else. I'll see what I can come up with. Will post here and update the PR as soon as I've got a working solution.

**Comment by tomhatzer** [comment]
@freekmurze this PR is nonsense, please ignore it. I'm going to open a new one with working code... Sorry for causing trouble.

**Pull request closed without merging**